kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/configuration/Password.java

```diff
@@ -33,6 +33,10 @@ public Password(char[] password) {
 
     @Override
     public String toString() {
+        if (this.m_password == null) {
```

Comment:
OK, it's a weird corner case and it does change the API. I'll drop this one and (c) can remain as is.